R/load.R

```diff
@@ -390,10 +537,11 @@ dotList <- function(input, spatial) {
     }
     recipient <- list(
       neighbours = unique(c(auxA$B, auxB$A, names(parallel)[!is.na(parallel)])),
-      before     = if (nrow(auxB) == 0) { NULL  } else { unique(auxB$A) },
-      after      = if (nrow(auxA) == 0) { NULL  } else { unique(auxA$B) },
-      parallel   = if ( !par.trigger  ) { NULL  } else { unique(names(parallel)[!is.na(parallel)]) },
-      edge       = if (nrow(auxA) == 0) { FALSE } else { any(auxA$Edge) })
+      before     = ifelse(nrow(auxB) == 0, NULL, unique(auxB$A)),
+      after      = ifelse(nrow(auxA) == 0, NULL, unique(auxA$B)),
+      parallel   = ifelse(!par.trigger, NULL, 
+                          unique(names(parallel)[!is.na(parallel)])),
+      edge       = ifelse(nrow(auxA) == 0, FALSE, any(auxA$Edge)))
```

Comment:
Apparently ifelse doesn't like it when one of the possible answers is `NULL`. I have a feeling I've been here before.

```suggestion
      before     = if (nrow(auxB) == 0) {
                     NULL 
                   } else {
                     unique(auxB$A)
                   },
      after      = if (nrow(auxA) == 0) {
                     NULL
                   } else {
                     unique(auxA$B)
                   },
      parallel   = if (!par.trigger) {
                     NULL
                   } else {
                     unique(names(parallel)[!is.na(parallel)])
                   },
      edge       = if (nrow(auxA) == 0) {
                     FALSE
                   } else {
                     any(auxA$Edge)
                   }
      )
```